modules/dlm/qa/with-security/build.gradle

```
@@ -0,0 +1,17 @@
+apply plugin: 'elasticsearch.legacy-java-rest-test'
+apply plugin: 'elasticsearch.authenticated-testclusters'
+
+dependencies {
+  javaRestTestImplementation project(":client:rest-high-level")
+}
+
+testClusters.configureEach {
+  testDistribution = 'DEFAULT'
+  setting 'xpack.watcher.enabled', 'false'
+  setting 'xpack.watcher.enabled', 'false'
+  setting 'xpack.ml.enabled', 'false'
+  setting 'xpack.license.self_generated.type', 'trial'
+  rolesFile file('roles.yml')
+  user username: "test_dlm", password: "x-pack-test-password", role: "manage_dlm"
+  user username: "test_non_privileged", password: "x-pack-test-password", role: "not_privileged"
```

To account for the privilege sitting behind a feature flag, I think we need:

```suggestion
  user username: "test_non_privileged", password: "x-pack-test-password", role: "not_privileged"
  requiresFeature 'es.dlm_feature_flag_enabled', Version.fromString("8.9.0")
```